examples/lang/erc20/src/lib.rs

```diff
@@ -218,20 +216,20 @@ mod tests {
 
     #[test]
     fn events_work() {
-        let alice = AccountId::try_from([0x0; 32]).unwrap();
-        let bob = AccountId::try_from([0x1; 32]).unwrap();
+        let alice: AccountId = From::from([0x0; 32]);
+        let bob: AccountId = From::from([0x1; 32]);
 
         // No events to start
-        env::test::set_caller(alice);
-        assert_eq!(env::test::emitted_events().count(), 0);
+        env::set_caller(alice);
+        assert_eq!(env::emitted_events().into_iter().count(), 0);
```

Comment:
imo `emitted_events` should continue to return an iterator

Reply:
Yes this is bypassing the `env::test::emitted_events` function. Need to figure this out. #125.